inc/compatibility/block-patterns/three-columns-images-texts-content.php

```diff
@@ -45,12 +49,16 @@
 <div class="wp-block-buttons"><!-- wp:button {"className":"is-style-primary"} -->
 <div class="wp-block-button is-style-primary"><a class="wp-block-button__link">LEARN MORE</a></div>
 <!-- /wp:button --></div>
-<!-- /wp:buttons --></div>
+<!-- /wp:buttons -->
+
+<!-- wp:spacer {"height":30} -->
+<div style="height:30px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer --></div>
 <!-- /wp:column -->
 
 <!-- wp:column -->
-<div class="wp-block-column"><!-- wp:image {} -->
-<figure class="wp-block-image "><img src="' . trailingslashit( get_template_directory_uri() ) . 'assets/img/patterns/neve-patterns-11.jpg" alt="" /></figure>
+<div class="wp-block-column"><!-- wp:image {"id":406,"sizeSlug":"large"} -->
+<figure class="wp-block-image size-large"><img src="' . trailingslashit( get_template_directory_uri() ) . 'assets/img/patterns/neve-patterns-11.jpg" alt="" class="wp-image-406"/></figure>
```

Comment:
Please remove `size-` and `wp-image-`class, as we meta attributes for image block, like `{"id":406,"sizeSlug":"large"}` for each image block. 

Like I said in Slack also, on import this will show an error because the ID of the attachment will not exist on imported site.

Reply:
I fixed and tested them and everything imports as it should.